### PR TITLE
Simplified sample. [API-1977]

### PR DESCRIPTION
**Pull request by emreyigit**

### CODE REVIEW

File: client_with_ssl.js [hunk 1]
```diff
@@ -1,305 +1,158 @@
-'use strict';
-
-const {Client, HazelcastJsonValue} = require('hazelcast-client');
-const fs = require('fs');
-const path = require('path');
-
-
-// This example shows how to work with Hazelcast maps.
-// @param client - a {@link HazelcastClient} client.
+// Copyright (c) 2008-2023, Hazelcast, Inc. All Rights Reserved.
```
Comment (srknzl) [inline client_with_ssl.js:1]: Can we add this to client.js as well?

Comment (srknzl) [reply client_with_ssl.js:1]: Why didn't you modified client.js file?

**Review by srknzl — COMMENTED** [review]
Other than client.js not modified, it looks good. You tried to run the sample right?

File: client_with_ssl.js (hunk 1 above)
Comment (emreyigit) [reply client_with_ssl.js:1]: client.js is not used but after make it ruining the the sample I can update. I get the map but still having problem with schemas. Do you have any suggestion? Pushed changes at https://github.com/hazelcast/hazelcast-cloud-nodejs-sample-client/pull/20/commits/d1a77886785820dc196122075974dd89a2a81494

File: client_with_ssl.js [hunk 2]
```diff
@@ -73,12 +72,16 @@ async function fetchCities(client) {
 
     console.log("OK.");
     console.log("--Results of 'SELECT __key, this FROM cities'");
-    printf("| %4s | %20s | %20s | %15s |\n", "id", "country", "city", "population");
+    sprintf("| %4$s | %20$s | %20$s | %15$s |\n", "id", "country", "city", "population");
+
+    // NodeJS client does lazy deserialization. In order to update schema table on the client,
+    // it's required to get map. 
+    const cities = await client.getMap("cities");
```
Comment (srknzl) [inline client_with_ssl.js:79]: a map.get() is required, not getMap

File: client_with_ssl.js [hunk 3]
```diff
@@ -127,25 +130,27 @@ class CitySerializer {
...
                             key: [fs.readFileSync(path.resolve(path.join(__dirname, 'key.pem')))],
-                            passphrase: 'YOUR_CLUSTER_PASSWORD',
+                            passphrase: 'd74f035016d',
                             checkServerIdentity: () => null
                         }
                     }
                 },
-                clusterName: 'YOUR_CLUSTER_NAME',                
+                clusterName: 'pr-bpmur7wb',                
                 serialization: {
```
Comment (srknzl) [inline client_with_ssl.js]: 🤔

Comment (emreyigit) [reply client_with_ssl.js]: I've removed them, sorry for the inconvenience.

**Comment by emreyigit** [comment]
Everything works now, tested against to Viridian prd.

File: client_with_ssl.js (hunk 2 above)
Comment (emreyigit) [reply client_with_ssl.js:79]: Fixed at https://github.com/hazelcast/hazelcast-cloud-nodejs-sample-client/pull/20/commits/d1a77886785820dc196122075974dd89a2a81494

File: client_with_ssl.js (hunk 1 above)
Comment (emreyigit) [reply client_with_ssl.js:1]: Fixed at https://github.com/hazelcast/hazelcast-cloud-nodejs-sample-client/pull/20/commits/91e1aba889e376e8c07e37459eb7f2556bce362a

**Review by srknzl — APPROVED** [review]

**Pull request merged**